src/platform/w3dengine/client/w3dview.cpp

```diff
@@ -38,6 +38,57 @@
 
 static const int g_theW3DFrameLengthInMsec = 33;
 
+W3DView::W3DView() :
+    m_3DCamera(nullptr),
+    m_2DCamera(nullptr),
+    m_viewFilterMode(FM_13),
+    m_viewFilter(FT_VIEW_SCREEN_DEFAULT_FILTER),
+    m_extraPass(false),
+    m_wireframeMode(false),
+    m_shakeAngleCos(0.0f),
+    m_shakeAngleSin(0.0f),
+    m_shakeIntensity(0.0f),
+    m_cameraShakeAngles(0.0f, 0.0f, 0.0f),
+    m_doingRotateCamera(false),
+    m_doingPitchCamera(false),
+    m_doingZoomCamera(false),
+    m_doingScriptedCameraLock(false),
+    m_FXPitch(1.0f),
+    m_doingMoveCameraOnWaypointPath(false),
+    m_doingMoveCameraAlongWaypointPath(false),
+    m_freezeTimeForCameraMovement(false),
+    m_timeMultiplier(0),
+    m_cameraHasMovedSinceRequest(true),
+    m_scrollAmountCutoff(0.0f),
+    m_groundLevel(0.0f),
+    m_cameraConstraintValid(false),
+    m_cameraSlaveMode(false),
+    m_realZoom(false)
+{
+    m_cameraOffset.z = g_theWriteableGlobalData->m_cameraHeight;
+    m_cameraOffset.y = -(m_cameraOffset.z / GameMath::Tan(g_theWriteableGlobalData->m_cameraPitch * 0.01745329300562541f));
+    m_cameraOffset.x = -(GameMath::Tan(g_theWriteableGlobalData->m_cameraYaw * 0.01745329300562541f) * m_cameraOffset.y);
+    m_shakeOffset.x = 0.0f;
+    m_shakeOffset.y = 0.0f;
+    m_locationRequests.clear();
+    m_locationRequests.reserve(50);
+    m_previousLookAtPosition.x = 0.0f;
+    m_previousLookAtPosition.y = 0.0f;
+    m_previousLookAtPosition.z = 0.0f;
+    m_scrollAmount.x = 0.0f;
+    m_scrollAmount.y = 0.0f;
+    m_cameraConstraint.hi.x = 0.0f;
+    m_cameraConstraint.hi.y = 0.0f;
+    m_cameraConstraint.lo.x = 0.0f;
+    m_cameraConstraint.lo.y = 0.0f;
```

Comment:
What about `m_rcInfo`, `m_pcInfo`, `m_zcInfo`, `m_mcwpInfo`? They are not fully initialized.